web-common/src/features/alerts/delivery-tab/AlertDialogDeliveryTab.svelte

```diff
@@ -14,6 +15,36 @@
     description="We'll check for this alert whenever the data refreshes"
     title="Schedule"
   />
+  <FormSection
+    description="Choose the interval at which the alert is evaluated since the prior data refresh."
+    title="Evaluation interval"
+    tooltip
+  >
+    <Select
+      bind:value={$form["splitByTimeGrain"]}
+      id="splitByTimeGrain"
```

Comment:
Ditto as mentioned above – the form field should be `interval` or `evaluationInterval`